backend/algorithms/objects/conditions.py

```diff
@@ -434,3 +460,10 @@ def __str__(self, id='start') -> str:
                 data['children'].append(json.loads(str(cond)))
         return json.dumps(data)
 
+def flatten_list_list(nested_list: List[List[str]]) -> List[str]:
```

Comment:
i dont think you need to use this function anymore if you use my list comprehension above.